coalib/output/ConsoleInteraction.py

```diff
@@ -864,20 +864,19 @@ def show_enumeration(console_printer,
 def show_bear(bear,
               show_description,
               show_params,
-              console_printer):
+              console_printer,
+              args=None):
```

Comment:
If we are getting `args` as param, we don't need `show_description` and `show_params`. I've kept these not to break API.

Reply:
Hm those are positional arguments occurring in the middle... could you try to deprecate those somehow? :)

Reply:
oh but just seeing, those arguments are still in use.